libs/orders/src/lib/components/order-list/order-list.spec.tsx

```diff
@@ -178,10 +178,12 @@ describe('OrderListTable', () => {
       });
 
       const amendCell = getAmendCell();
+      const typeCell = screen.getAllByRole('gridcell')[2];
+      expect(typeCell).toHaveTextContent('Liquidity provision');
       expect(amendCell.queryAllByRole('button')).toHaveLength(0);
     });
 
-    it('doesnt show buttons for pegged orders', async () => {
+    it('does not show buttons for pegged orders', async () => {
```

Comment:
suggestion (non blocking):
```suggestion
    it('shows if an order is a pegged order and does not show order actions', async () => {
```